benchmark/ngu_formula.py

```diff
@@ -0,0 +1,52 @@
+"""
+NGU alpha formula.
+
+Idea:
+    The more the instruments fell, the more we buy.
+"""
+
+from thousandaire.alpha import BaseAlphaFormula
+from thousandaire.data_classes import Portfolio
+
+def make_list(multi, days):
+    """
+    Make position list.
+    """
+    position = [0, 1]
+    count = days
+    while count:
```

Comment:
I'm not sure about this, but I thought negative numbers are also true?

Reply:
Yeah, negative numbers are also true.  I think that will not be negative because the days we pass always be positive.

Reply:
@ioumu8 While I know that there won't be negative numbers, I agree with shervyna that we can make it more specific. For example, `while count > 0` may be better.

Reply:
umm, I mean in this while loop you did ```count -=1``` on line 19, but the condition for this while loop is ```while count```.  Negative numbers are also true. Then isn't this loop an infinite loop...?

Reply:
In line 16 ```count = days ``` and  ```days``` always > 0 , so this loop would not be an infinite loop, even used ```count -= 1``` (Before count < 0, count would be equal to 0).
However, it's true that I should make it more specific, so I will change it as b821213's comment.

Reply:
oh i see sorry I forgot the zero :D thanks